src/matrix.jl

```diff
@@ -1,10 +1,10 @@
-istable(::AbstractMatrix) = false
-istable(::Type{<:AbstractMatrix}) = false
+istable(::AbstractVecOrMat) = false
+istable(::Type{<:AbstractVecOrMat}) = false
 
-rows(m::T) where {T <: AbstractMatrix} = throw(ArgumentError("a '$T' is not a table; see `?Tables.table` for ways to treat an AbstractMatrix as a table"))
-columns(m::T) where {T <: AbstractMatrix} = throw(ArgumentError("a '$T' is not a table; see `?Tables.table` for ways to treat an AbstractMatrix as a table"))
+rows(m::T) where {T <: AbstractVecOrMat} = throw(ArgumentError("a '$T' is not a table; see `?Tables.table` for ways to treat an AbstractVecOrMat as a table"))
+columns(m::T) where {T <: AbstractVecOrMat} = throw(ArgumentError("a '$T' is not a table; see `?Tables.table` for ways to treat an AbstractVecOrMat as a table"))
```

Comment:
I think the changes for `rows` and `columns` are breaking and `istable` already returned `false` AFAICT. As commented earlier we currently allow:
```
julia> Tables.columns([(1,2),(3,4)])
Tables.CopiedColumns{NamedTuple{(Symbol("1"), Symbol("2")), Tuple{Vector{Int64}, Vector{Int64}}}} with 2 rows, 2 columns, and schema:
 Symbol("1")  Int64
 Symbol("2")  Int64
```

Reply:
I reverted the changes to these functions.